website/docs/jellyfish/api/poolpair.md

```diff
@@ -160,3 +160,48 @@ interface PoolShareOptions {
   isMineOnly?: boolean
 }
 ```
+
+## poolSwap 
+
+Creates a poolswap transaction with given metadata.
+
+```ts title="client.poolpair.poolSwap()"
+interface PoolSwap {
+  poolSwap (metadata: PoolSwapMetadata, inputs?: PoolSwapInputs): Promise<string>
+}
+
+interface PoolSwapMetadata {
+  from: string
+  tokenFrom: string
+  amountFrom: number
+  to: string
+  tokenTo: string
+  maxPrice: number
+}
+
+interface PoolSwapInputs {
```

Comment:
PoolSwapInputs is a decent name. But for this variable, we declare like below in other typescript file

```
export interface UTXO {
  txid: string
  vout: number
}
```